cypress/support/cypress-support/src/main.js

```diff
@@ -55,6 +55,22 @@ export default function (module) {
       } else {
         cy.log('Unable to establish a pub/sub connection.');
       }
+      const topic = UTILS.getTopic(
+        UTILS.getEnvVariable(CONSTANTS.FIRST_PARTY_APPID),
+        CONSTANTS.SUBSCRIBE,
+        null,
+        CONSTANTS.TOPIC_FBINTERACTIONS
```

Comment:
I am not sure I understood line 65 . what is that trying to do >?

Reply:
initiating the Interaction service to get the logs